packages/core/src/card/card.element.scss

```diff
@@ -18,3 +18,18 @@
     --width: -moz-fit-content;
   }
 }
+
+:host-context([cds-layout~='grid'][cds-layout*='align:horizontal-stretch']) {
```

Comment:
This is the only place in the code base that uses host-context, so I'm not positive this is the correct approach?

Reply:
This is because host-context does not have wide enough [browser support](https://caniuse.com/?search=%3Ahost-context()). We also probably would want take a different approach since this ties the implementation to specifically cds-layout. The card bug still happens if the card is placed in a standard CSS grid layout.

I think the fix might be to allow the card to fill its container by setting the width and height custom properties to default to 100% rather than fit-content and auto. This would align card to the rest of our block component behavior like alerts/forms/tables. This is slightly a breaking change from the default behavior (which is fine for this PR on 6.0/next).